### ISSUE TITLE
VirtualBox error

### ISSUE THREAD

**Issue by mrneilypops** [MarkHedleyJones/dmenu-extended#70]
Hi Mark,
Here I am again developing my little distro dwmX.
I have an issue with dmenu_extended when running dwmX in a VirtualBox...see attached screenshot.
This does not happen on bare metal.

Any ideas much appreciated.

https://dwmx.org
https://sourceforge.net/projects/dwmx/
![screenshot from 2018-10-30 14-54-19](https://user-images.githubusercontent.com/1450448/47722903-b3b24080-dc53-11e8-960f-78f1746c031a.png)

**Comment by MarkHedleyJones** [comment]
Hi Niel,
Great to see your getting back into your distro.
I've not come across that error before.
Could you try deleting all .pyc files under the dmenu_extended install location and trying again?

**Comment by mrneilypops** [comment]
If I comment out;
'import plugins' in /usr/local/lib/python2.7/dist-packages/dmenu_extended.py then dmenu_extended runs.
I will try an ISO build removing plugins and try and establish which one is causing the problem...

**Comment by mrneilypops** [comment]
![screenshot from 2018-10-31 15-02-31](https://user-images.githubusercontent.com/1450448/47799471-9e5c1580-dd2a-11e8-8831-8390b8cff398.png)

dmenu-extended runs on live session in VirtualBox.
dmenu-extended has the above error after install in VirtualBox.
'sudo dmenu-extended' works after install...

**Comment by MarkHedleyJones** [comment]
If you now remove all .pyc files related to dmenu_extended in your post
installation environment does it run then?

**Comment by mrneilypops** [comment]
No. Removing the .pyc file did not work. I had to purge the existing installation and reinstall to get dmenu_extended running again.

**Comment by MarkHedleyJones** [comment]
Which .pyc file did you remove exactly? My money would be on the one located at `~/.config/dmenu-extended/plugins/__init__.pyc`
